README.md

```diff
@@ -50,7 +50,6 @@ require'lspconfig'.cssmodules_ls.setup {
 ```
 
 ### [coc.nvim](https://github.com/neoclide/coc.nvim)
-
```

Comment:
This change seem unrelated

Reply:
Reverted this in new commit, thanks for considering merging!